<ui version="4.0" >
 <class>uorSheetPanel</class>
 <widget class="QWidget" name="uorSheetPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="horizontalSpacing" >
      <number>1</number>
     </property>
     <property name="verticalSpacing" >
      <number>1</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="m_labAdress" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>47</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Adress:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="indent" >
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="m_AdressCellEdit" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>94</width>
         <height>200</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="maxLength" >
        <number>60</number>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="m_labFind" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>42</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>42</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Find:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="indent" >
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QLineEdit" name="m_FindEdit" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>103</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>103</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text" >
        <string>&lt;Не пашет пока></string>
       </property>
       <property name="maxLength" >
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QToolButton" name="m_findDown" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="../uoReport.qrc" >:/image/SearchDown.png</iconset>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5" >
      <widget class="QToolButton" name="m_findUp" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="../uoReport.qrc" >:/image/SearchUp.png</iconset>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6" >
      <widget class="QLabel" name="m_text" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>29</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>29</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Text</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="7" >
      <widget class="QLineEdit" name="m_cellText" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="cursor" >
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="dragEnabled" >
        <bool>false</bool>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../uoReport.qrc" />
 </resources>
 <connections/>
</ui>
